### ISSUE TITLE
Swiss QR-Bill - Reminder never prints

### ISSUE THREAD

**Issue by DominicSaladin** [microsoft/ALAppExtensions#21335]
We are currently implementing the qr code into the reminder for our customer.
For this, we created a custom report for the reminder and used (and copied some parts of it) the QR Management.
But unfortunately the Swiss QR-Bill module did implement the reminder wrong.

Inside `Print.Report.al` the dataitem for the Issued Reminder Header calls `FilterCustLedgerEntry` with the document type Reminder. Problem is, if you issue a reminder, no customer ledger entry gets inserted, but a Reminder/Fin. Charge Entry. So if you print the reminder qr, the report never prints out a qr code.
How is it possible that no one reported this? This is a major bug in this module.

**Comment by AlexanderYakunin** [comment]
As we are working with extensibility requests here, we cannot help you to address this issue. Should you be the first one who create hotfix requests for this issue via technical support?

**Comment by DominicSaladin** [comment]
I'm sorry, but I don't quite understand your question. Could you rephrase your question?

**Comment by AlexanderYakunin** [comment]
Hi! Please take a minute to read the guidelines for this repository https://github.com/microsoft/AL and our collection of frequently asked questions https://github.com/Microsoft/AL/wiki/frequently-Asked-Questions .
This repository is only for extensibility requests for Dynamics 365 Business Central. We suggest that you open a support case or file a bug to ensure that all Business Central users benefit from your catch as soon as possible:
•	On premises - https://support.microsoft.com/en-us/supportforbusiness/productselection?sapId=93d37907-ad94-d591-22e9-593cfa09dd3f
•	On line – contact your partner, or if you are a partner open a support incident through Partner Center - https://partner.microsoft.com/en-us/pcv/dashboard/overview
